docs/api/resolving_proxies.rst

```diff
@@ -0,0 +1,35 @@
+Resolving Proxies
```

Comment:
I'd name it something like `Proxy resolvers` or alike. `Resolving proxies` is more fitting for guide section than for api reference.